File: app/src/main/java/com/example/finances/ui/Account/Course.java
package com.example.finances.ui.Account;


public class Course {

    private String name; // название
    private String capital;  // столица
    private int flagResource; // ресурс флага

    public Course(String name, String capital, int flag){

        this.name=name;
        this.capital=capital;
        this.flagResource=flag;
    }

    public String getName() {
        return this.name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCapital() {
        return this.capital;
    }

    public void setCapital(String capital) {
        this.capital = capital;
    }

    public int getFlagResource() {
        return this.flagResource;
    }

    public void setFlagResource(int flagResource) {
        this.flagResource = flagResource;
    }
}

File: app/src/main/java/com/example/finances/database/Course.kt
package com.example.finances.database
class Course (
    var id: Int = 0,
    var name: String? = null,
    var startDate: Long = 0,
    var endDate: Long = 0,
    var finished: Int = 0,
    var lessons: Int = 0,
    var lessonsCompleted: Int = 0)